src/pages/projectSettings/tabs/NotificationsTab/types.ts

```diff
@@ -1,42 +1,61 @@
+import { Subset, Unpacked } from "types/utils";
 import { ProjectType } from "../utils/types";
 
+export type Notification = Subset<{
```

Comment:
Would `Partial` be more appropriate than `Subset` here? It looks to me like `Subset` is recursive, and I think we'd rather explicitly define the shape of the nested objects than make them optional via recursion.